src/Tickets/Commerce/Flag_Actions/Send_Email.php

```diff
@@ -34,6 +34,14 @@ class Send_Email extends Flag_Action_Abstract {
 	 * {@inheritDoc}
 	 */
 	public function handle( Status_Interface $new_status, $old_status, \WP_Post $order ) {
+
+		// temporary fix for manual attendees first email
+		// @todo backend review this logic
+		if ( ! empty( $order->gateway ) && 'manual' === $order->gateway && empty( $order->events_in_order ) ) {
+			$order->events_in_order[] = $order;
```

Comment:
Why are we adding the order to the events array?

Reply:
This is a dirty workaround.

That said, it works because when adding attendees manually the Order already contains all the necessary information to send the email. In this case, Order ID and Event ID are identical when calling `send_tickets_email` but all the email information is re-fetched from the Order later, including which ticket and event it belongs to, so all information coming out in the email is correct.

Reply:
Ohh I see. Thansk a lot